net-misc/urlscan/urlscan-0.9.1.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_{4,5,6,7} )
```

Comment:
Sorry it's taken forever, but please remove support python3.4 and rebase against fresh copy of master repo.